html/index.html

```diff
@@ -28,7 +28,7 @@ <h1>AngularJS and Leaflet Exploration</h1>
           </div>
 
           <div class="col-md-6">
-            <h2>TODO List</h2>
+             <h2>TODO list({{counter}} tasks)</h2>
```

Comment:
It looks like you added a space before `<h2>` that doesn't belong there.

Should there also be a space after "list"? So that it says "TODO list (1 tasks)" instead of "TODO list(1 tasks)".

Can you make it so that it says "TODO list (1 task)" instead of "TODO list (1 tasks)"? But still keep "TODO list (2 tasks)"? A little challenge for you 😄